hassio/addon-view/hassio-addon-logs.html

```diff
@@ -11,6 +11,9 @@
       paper-card {
         display: block;
       }
+      pre {
+        overflow-x: scroll;
```

Comment:
We should never set `overflow` to be scroll. This means that we **always** show a scrollbar on Windows.